core/internal/settings/settings.go

```diff
@@ -257,3 +263,39 @@ func (s *Settings) GetIdentityTokenFile() string {
 func (s *Settings) IsConsoleCaptureEnabled() bool {
 	return s.Proto.Console.GetValue() != "off"
 }
+
+// The program path that created the run.
```

Comment:
```suggestion
// Path to the script that created the run, if available.
```

(correct me if I'm wrong)

Reply:
that's exactly right! thanks!